locations/paas/src/main/java/brooklyn/location/PaasProvisioningLocation.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.location;
+
+public interface PaasProvisioningLocation<T extends PaasLocation> extends ProvisioningLocation<T> {
```

Comment:
@ mbarrientos I think we will not use the CLI, but [cf-client-lib](https://github.com/cloudfoundry/cf-java-client/tree/master/cloudfoundry-client-lib)

@kiuby88 it is not so similar, the `space` returned by a PaaS is technologically a container, but probably we can't always ssh into it, so how can extends `SshMachineLocation` ?